packages/api/src/util/generateFinanceReport.js

```diff
@@ -0,0 +1,15 @@
+import ExcelJS from 'exceljs';
+
+const generateFinanceReport = (data) => {
```

Comment:
Not really necessary to address this comment now... but I am thinking it could be beneficial to separate the creation of the workbook from the making of the "Sheet 1".  Such that when you pass in a workbook to a sheet making function you can add sheets (or not). If in the future we have multiple reports like "balance sheet", "transactions", "expense report", "tax report" etc. the user might check off which ones to generate and we just pass along the workbook to each?

Just a thought.

Reply:
Yeah, I could totally see passing the workbook to a different function for each sheet type here! I don't think using sheets in that way has been discussed yet (as opposed to just creating different kinds of reports downloads entirely) but it's something to bring up with product when we get there and I think it makes sense. It's definitely technically feasible.